config/settings.yml

```diff
@@ -2,3 +2,4 @@
 features:
   make_question_optional: true
   task_list_statuses: false
+  submission_email_confirmation: false
```

Comment:
Could set this to `true` unless we're waiting on other work or want to do this in a separate PR

Reply:
I think we need to wait until the Notify integration and end-2-end tests are in place.

Reply:
Makes sense!